vermouth/data/force_fields/martini3001/aminoacids.ff

```diff
@@ -391,7 +635,53 @@ HIH                1
 #meta {"group": "Side chain bonds", "ifdef": "FLEXIBLE"}
 ;  i     j   funct   length  
  SC1   SC2    1       0.320  $stiff_fc
- SC1   SC3    1       0.300  $stiff_fc 
+ SC1   SC3    1       0.320  $stiff_fc 
```

Comment:
intended?

Reply:
As mentioned in the answer below. For any protonated HIS models, SC1-SC2 and SC1-SC3 bond lengths should be 0.32 nm.